lib/loomio/loomio_handler.rb

```diff
@@ -0,0 +1,73 @@
+require 'httparty'
+
+class LoomioHandler
+  include HTTParty
+  @@base_uri = 'https://talk.theborderland.se/'
```

Comment:
This should go into an ENV so that other people can use it with their Loomio instances if they wish.